lib/xcodeproj/constants.rb

```diff
@@ -382,5 +382,12 @@ module Constants
     # @return [Array] The extensions which are associated with header files.
     #
     HEADER_FILES_EXTENSIONS = %w(.h .hh .hpp .ipp .tpp .hxx .def).freeze
+
+    # @return [Array] The keywords Xcode use to identify a build setting can
+    #                 inherit values from a previous precedence level
+    INHERITED_KEYWORDS = %w(
```

Comment:
@samdmarshall do you know if this encompasses all of the ways of spelling `inherited` in build settings?